cromwell.example.backends/PAPIv2.conf

```diff
@@ -57,6 +57,10 @@ backend {
         #  auth = "application-default"
         # }
 
+        # Global pipeline timeout
+        # Defaults to 7 days
+        # pipeline-timeout = 3 days
```

Comment:
One convention I quite like is to make the commented-out value match whatever the "not supplied" default is.

So in other words, I'd make this:
```
        # Timeout to pass to GCE when starting jobs ():
        # pipeline-timeout = 7 days
```

Reply:
+1

Reply:
Updated!